alf/networks/relu_mlp.py

```diff
@@ -134,8 +137,11 @@ def compute_jac(self, inputs):
             ("inputs should has shape {}!".format(self._input_size))
 
         self.forward(inputs)
+        J = self._compute_jac()
+        if inputs.ndim == 1:
+            J = J.squeeze()
```

Comment:
To be safe, use J.squeeze(0) in case there are some other singleton dimensions,

Reply:
Updated here and other places.